generators/client/templates/angular/src/test/javascript/jest-global-mocks.ts.ejs

```diff
@@ -1,15 +1,3 @@
-const mock = () => {
-    let storage = {};
-    return {
-        getItem: key => (key in storage ? storage[key] : null),
-        setItem: (key, value) => (storage[key] = value || ''),
-        removeItem: key => delete storage[key],
-        clear: () => (storage = {})
-    };
-};
-
-Object.defineProperty(window, 'localStorage', { value: mock() });
-Object.defineProperty(window, 'sessionStorage', { value: mock() });
```

Comment:
why removing this? Are they mocked by jest himself?

Reply:
`jsdom` supports web storage APIs, so, no need to mock.

Reply:
Oh ok, perfect then :)